### PR TITLE
Documentation: Fix small typos and grammatical mistakes #4321

### LINKED ISSUE #4321: Fix small typos and grammatical errors
Motivation
----------
Small documentation patch for simple typos and grammatical errors.

Modification
------------
Fixed typos in  /bin/rucio-hermes2 , /bin/rucio-conveyor-throttler, /bin/rucio-conveyor-fts-throttler, /bin/rucio-necromancer

### PR DESCRIPTION
**Pull request by OCM-7898**
Fixed typos and grammatical mistakes in  bin/rucio-conveyor-fts-throttler,bin/rucio-conveyor-throttler,bin/rucio-hermes2,bin/rucio-necromancer

### CODE REVIEW

File: bin/rucio-conveyor-fts-throttler [hunk 1]
```diff
@@ -19,7 +19,7 @@
 """
 Conveyor FTS Throttler is a daemon that will configure a fts storage's transfer settings
 depending on how many time out errors occur at the storage. If a storage has substantial amount
-of transer failures due to time outs, it is usually due to bad connectivity and the amount of failures can be
+of tranfser failures due to time outs, it is usually due to bad connectivity and the amount of failures can be
```
Comment (bziemons) [inline bin/rucio-conveyor-fts-throttler]: almost…!

**Review by bziemons — CHANGES_REQUESTED** [review]

File: bin/rucio-conveyor-fts-throttler (hunk 1 above)
Comment (OCM-7898) [reply bin/rucio-conveyor-fts-throttler]: Hopefully it's fixed now

**Review by bziemons — APPROVED** [review]
Thank you very much! :)

**Comment by bari12** [comment]
👍

**Pull request merged**